### PR TITLE
HPCC-27306 Add limits for httptest times/threads

### PR DESCRIPTION
**Pull request by wangkx**
Signed-off-by: wangkx <kevin.wang@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by wangkx** [comment]
@AttilaVamos this PR is for fixing the Coverity Scan defect. How to run the Coverity Scan to verify the fix or could you please verify it for me?

File: esp/test/httptest/main.cpp [hunk 1]
```diff
@@ -17,6 +17,9 @@
 
 #include "httptest.hpp"
 
+constexpr unsigned TIMES_LIMIT = 10000000; //10M
```
Comment (afishbeck) [inline esp/test/httptest/main.cpp:20]: @AttilaVamos Can you comment.  I'm not sure where httptest is used to know what these limits should be used.  It's just a parameter check.  If httptest is meant to be able to do long stress testing I could see high values being desirable.  Just not sure what values.

Comment (AttilaVamos) [reply esp/test/httptest/main.cpp:20]: @afishbeck The Coverity scan is a static analysis. If the httptest is a stand-alone test and can run long-term (how and how long?) then I can add it to OBT.

@wangkx The Coverity scan executed once a week (limitation of free scan) on Monday with master branch.

**Review by afishbeck — APPROVED** [review]

**Review by AttilaVamos — APPROVED** [review]
It seems ok.

**Pull request merged**